src/main/java/com/bulkbuy/enterprise/dao/OrderDAOStub.java

```diff
@@ -1,4 +1,7 @@
 package com.bulkbuy.enterprise.dao;
 
 public class OrderDAOStub {
+    public void saveOrder(){
+
```

Comment:
I actually meant to delete this.  Please ignore

Reply:
Then delete it in a commit and push.  You can re open a pull request.   Unused code is clutter, which is technical debt.